backend/src/database/migrations/tests/is-credential-used.test.ts

```diff
@@ -0,0 +1,198 @@
+import { Sequelize } from 'sequelize-typescript'
```

Comment:
wow this looks cool! this is like the first db function test in our codebase.

wonder if there is a better place to put it rather than the `tests` folder